backend/Sources/backend/Controllers/MeetupController.swift

```diff
@@ -0,0 +1,165 @@
+//
+//  MeetupController.swift
+//  backend
+//
+//  Created by Mauricio Cardozo on 9/28/25.
+//
+
+import CocoaHeadsCore
+import SwiftSoup
+import Vapor
+
+struct MeetupController: RouteCollection {
+  func boot(routes: any Vapor.RoutesBuilder) throws {
+    let meetupRoutes = routes.grouped("meetup")
+    meetupRoutes.post(use: self.event)
+  }
+
+  @Sendable
+  func event(req: Request) async throws -> MeetupEvent {
+    let meetup = try req.content.decode(MeetupEventRequest.self)
+
+    guard
+      let url = URL(string: meetup.url),
+      case (let data, _) = try await URLSession.shared.data(from: url),
+      let html = String(data: data, encoding: .utf8)
+    else {
+      throw Error.urlError
+    }
+
+    guard !url.absoluteString.contains("Entrar") else {
```

Comment:
is "Entrar" a safe string to use?

Reply:
Used to be, but meetup has released a completely new design while I was working on moving this code around, so I'll need to do more testing to figure out if everything here still works